packages/snap/README.md

```diff
@@ -55,7 +58,65 @@ Once connected, you can call the RPC method exposed in this snap as follows:
 console.log(result);
   
 ```
-
 MetaMask will automatically fetch the Snap from the npm registry.
 
 Please refer to the documentation for more about the parameters field https://docs.metamask.io/snaps/reference/rpc-api/#example-1
+
+### More Generic Transaction Example
+This example shows how one can use the snap to send a more generic transaction.
+
+```javascript
+
+  import { CeloTransactionRequest } from '@celo-tools/celo-ethers-wrapper';
+
+  const abi = ['function deposit() external payable']
+  const iface = new ethers.utils.Interface(abi)
+  const data = iface.encodeFunctionData("deposit")
+
+  let tx: CeloTransactionRequest = {
+      from: <wallet_address>,
+      to: <destination_contract_address>,
+      data
+      value: ethers.utils.parseUnits("1", "wei")
+  }
+
+  const gasLimit = (await wallet.estimateGas(tx)).mul(10)
```

Comment:
I dont think we need to multiple the gas estimation by 10 anymore. The full nodes will give the proper amount

Reply:
Just removed this from the readme. User dose not need to pass it. The snap will do that job.
On that note i left it with the multiplied excess following the libraries [suggestion](https://github.com/celo-tools/celo-ethers-wrapper#alternative-gas-fee-currencies) - i'll need to do more testing to be confident that the full nodes will give proper amount but it does not hurt to pad it like so, since the transaction will only take the gas it needs - and fees are quite cheep in celo